Altis_Life.Altis/core/actions/fn_buyLicense.sqf

```diff
@@ -6,14 +6,13 @@
     Description:
     Called when purchasing a license. May need to be revised.
 */
-private ["_type","_varName","_displayName","_sideFlag","_price"];
-_type = _this select 3;
+params ["", "", "", "_type"];
```

Comment:
No can do for the index (has to be 4th). Could move it over to `param`, but for standard, `params` is being used where possible (can discuss)